src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,85 @@
+package lotto.controller;
+
+import lotto.constants.Grade;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lotto.domain.WinningInformation;
+import lotto.domain.Lotto;
+import lotto.domain.LottoNumber;
+import lotto.domain.Money;
+import lotto.domain.PointCalculator;
+import lotto.domain.PointResult;
+import lotto.utils.Parser;
+import lotto.utils.RandomUtils;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+    public void run() {
+        Money money = generateMoney();
+        PointResult pointResult = generatePointResult(money.getLottoPurchaseCount());
+        printLottoResult(money, pointResult);
+    }
+
+    private PointResult generatePointResult(int lottoPurchaseCount) {
+        List<Lotto> lottos = generateLottoWithCount(lottoPurchaseCount);
+        WinningInformation winningInformation = generateWinningInformation();
+        List<Double> lottoPointResult = getLottoPointResult(lottos, winningInformation);
+        return new PointResult(lottoPointResult);
+    }
+
+    private List<Lotto> generateLottoWithCount(int count) {
+        OutputView.printLottoPurchaseCount(count);
+        List<Lotto> lottos = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            lottos.add(generateLotto());
+        }
+        return lottos;
+    }
+
+    private Lotto generateLotto() {
+        List<LottoNumber> randomLottoNumbers = RandomUtils.generateRandomLottoNumber().stream()
+                .sorted()
+                .map(LottoNumber::of)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .map(LottoNumber::of)
                .toList();
```

자바 17에서 새로 적용된 문법을 제안드립니다!